Turns out the 47189 dts file did not include
generated dt binding header file. This meant none
of the kernel options were propagated to the
resulting dtb file.

This manifested eg. with broken (no ramoops were
generated) ramoops support on 47189 boards.
--- a/kernel/dts/47189/947189.dts
+++ b/kernel/dts/47189/947189.dts
@@ -1,3 +1,5 @@
+#include <dt-bindings/soc/bcm963xx_dt_bindings.h>
+
 /dts-v1/;
 
 / {
